test/app-luatest/gh_5747_crash_multiple_args_test.lua

```diff
@@ -3,7 +3,11 @@ local t = require('luatest')
 local g = t.group()
 
 g.test_multiple_args = function()
-    local handle = io.popen('tarantool -ea=10 -e b=20 -ec=30 -e d=40 -e"print(a+b+c+d)"')
+    local TARANTOOL_PATH = arg[-1]
+    local cmd =
```

Comment:
This alignment looks a little bit weird to me. I suggest something like the following:
```lua
    local cmd = ('%s -ea=10 -e b=20 -ec=30 -e d=40 -e"print(a+b+c+d)"'):format(
                 TARANTOOL_PATH
    )
```
or
```lua
    local cmd = string.format(
        '%s -ea=10 -e b=20 -ec=30 -e d=40 -e"print(a+b+c+d)"',
        TARANTOOL_PATH
    )
```
Feel free to ignore though.

Reply:
Updated (second variant):

```diff
--- a/test/app-luatest/gh_5747_crash_multiple_args_test.lua
+++ b/test/app-luatest/gh_5747_crash_multiple_args_test.lua
@@ -4,9 +4,10 @@ local g = t.group()
 
 g.test_multiple_args = function()
     local TARANTOOL_PATH = arg[-1]
-    local cmd =
-        ('%s -ea=10 -e b=20 -ec=30 -e d=40 -e"print(a+b+c+d)"'):
-        format(TARANTOOL_PATH)
+    local cmd = string.format(
+        '%s -ea=10 -e b=20 -ec=30 -e d=40 -e"print(a+b+c+d)"',
+        TARANTOOL_PATH
+    )
     local handle = io.popen(cmd)
     local result = handle:read()
     t.assert_equals(result, '100')
```